tests/typescript-project-runner.ts

```diff
@@ -54,7 +54,6 @@ export default class ProjectRunner {
 }
 
 export class Project {
-  public baselineDir: string;
```

Comment:
The issue here is not that the ts definition for project is incorrect, ts2.7.x has a strict class initialization feature to ensure that class properties are init according to their defined types: https://github.com/Microsoft/TypeScript/commit/148dc4e013983ecc71b6e317025e9250d8342d03

This attribute `baselineDir` is indeed used in broccoli tsc :https://github.com/tildeio/broccoli-typescript-compiler/blob/245af5a79fc87617d7a0675da8b3f4a03918ce33/tests/typescript-project-runner.ts#L92
 and is defined as an es5 getter: https://github.com/tildeio/broccoli-typescript-compiler/blob/245af5a79fc87617d7a0675da8b3f4a03918ce33/tests/typescript-project-runner.ts#L96

One way so tell tsc that this prop indeed exists is with a "definite assignment assertion", new in ts 2.7.x
https://github.com/Microsoft/TypeScript/commit/1624e1bc1ae64dfe741ee55af5c5556cf3d100b1

Reply:
oo nice 👍